src/Halcyon/Datasource/FileDatasource.php

```diff
@@ -325,19 +332,19 @@ protected function validateDirectoryForSave(string $dirName, string $fileName, s
      */
     protected function makeDirectoryPath($dirName, $relativePath = '')
     {
-        $base = $this->basePath . '/' . $dirName;
-        if (!empty($relativePath)) {
-            $path = $base . '/' . $relativePath;
-        } else {
-            $path = $base;
-        }
+        $base = $this->basePath . DIRECTORY_SEPARATOR . $dirName;
+        $path = !empty($relativePath) ? $base . DIRECTORY_SEPARATOR . $relativePath : $base;
```

Comment:
@jaxwilko I know your way is better, but given this class still uses forward-slashes throughout, `DIRECTORY_SEPARATOR` might give some inconsistent results.

```suggestion
        $base = $this->basePath . '/' . $dirName;
        $path = !empty($relativePath) ? $base . '/' . $relativePath : $base;
```

Reply:
Fair enough, i did wonder why the const wasn't being used, happy to change it back, was just doing some cleanup while i was there